app/views/posts/index.html.erb

```diff
@@ -6,9 +6,21 @@
       <%= post.title %>
     </li>
 <%= post.body %>
+<% if user_signed_in? %>
+<strong>
+<p> Authors name:
+<%= post.user.user_name %>
+</p>
+</strong>
+<% end %>
   <% end %>
 </ul>
```

Comment:
- [ ] Kindly remove all the logic from your views. This is important because the app will follow the MVC pattern. Kindly create the logic inside the helpers. Kindly use this [article](http://codefol.io/posts/Where-Do-I-Put-My-Code/) to learn more
NB: Logic includes: if statements and loops. ⬇️ 

<img width="825" alt="no_logic_in_helpers_requirement" src="https://user-images.githubusercontent.com/32361963/115119265-28a69b80-9fb0-11eb-8ea1-132706eb2cc2.png">